### DISCUSSION TITLE
Long response time in Next API routes

### DISCUSSION THREAD

**Discussion by aniquehussain** [vercel/next.js#48359] [Help]
### Problem Description

We started using next api routes for a couple of apis, but we're getting responses after 6-8 seconds, which is ruining the user experience. We spent a long time debugging the problem and discovered that the logic execution only takes 0.8s to 1.1s. We also created a new project with similar configuration and copied the api route to compare performance and discovered that the new project performs better and has a 40-50% shorter response time.

The only difference between the two projects is that the production project is using Next version "12.2.5" while the test project is using Next version "13.3.0." Please let me know if this is the problem.

I've attached two screenshots that show production and test results as well as response times. You can also see the total time spent on logic execution (latencyTotal).

I would greatly appreciate any help.

<img width="522" alt="Production" src="https://user-images.githubusercontent.com/47793113/231424339-af00cf73-768a-41b5-bf3d-aee29995ae17.png">

<img width="483" alt="Test" src="https://user-images.githubusercontent.com/47793113/231424432-ccfaa774-a76e-4160-a0e2-651a4896bcfe.png">

**Comment by wiiiimm** [comment]
@aniquehussain can you share your test repos for both Next v12 and v13 with us so I can take a closer look?
